### ISSUE TITLE
[Accessibility]A11y_VSCode_ShowOpenedEditors_ScreenReader:Voiceover does not read the search results found after typing any command in editor. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52326]
Environment Details:
VSCode Version : 1.23.1
OS Version : MAC OS HighSierra10.13.5
Additional Details:
MAS Violated : MAS 1.3.1
ScreenReader Versions : VoiceOver
Repro Steps:
Launch VS Code while Voiceover is ON.
1.Navigate to welcome screen and beside opened editors pane at top(in middle pane)->Select More(…)->Select "Show Opened Editors" from dropdown.
2.Navigate to editor and search for any commands.
Actual:
Voiceover  does not read the "No results found....." content which tells user the search result did not produce any valid result.
Expected:
JAWS /NVDA should read the "No results found...." content as soon as search is performed by user and no outputs are found.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
User Impact:
Screenreader users will not know about the empty search content and other meaningful information.When "No results...." content is read ,the users will then and there come to know about the search results output.
MAS Reference:
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Does this issue occur when all extensions are disabled?: Yes/No
Attachment for Reference:
[EditorSearch.pptx](https://github.com/Microsoft/vscode/files/2115149/EditorSearch.pptx)

**Comment by alvinkiett84** [comment]
Here are some times that work for me:


        Tuesday, June 19






4:30 AM - 5:00 AM (CDT)










Made easy by Outlook<http://aka.ms/Dcupce>








Get Outlook for Android<https://aka.ms/ghei36>



From: AccessibilityTestingTeam-TCS
Sent: Tuesday, June 19, 7:34 AM
Subject: [Microsoft/vscode] [Accessibility]A11y_VSCode_ShowOpenedEditors_ScreenReader:NVDA does not read the search results found after typinng any command in editor. (#52326)
To: Microsoft/vscode
Cc: Subscribed


Environment Details:
VSCode Version : 1.23.1
OS Version : MAC OS HighSierra10.13.5
Additional Details:
MAS Violated : MAS 1.3.1
ScreenReader Versions : VoiceOver
Repro Steps:
Launch VS Code while Voiceover is ON.
1.Navigate to welcome screen and beside opened editors pane at top(in middle pane)->Select More(…)->Select "Show Opened Editors" from dropdown.
2.Navigate to editor and search for any commands.
Actual:
Voiceover does not read the "No results found....." content which tells user the search result did not produce any valid result.
Expected:
JAWS /NVDA should read the "No results found...." content as soon as search is performed by user and no outputs are found.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

Get Outlook for Android<https://aka.ms/ghei36>

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51824
